src/team3.html

```diff
@@ -164,41 +159,15 @@
 		else if(language == "Turkish")
 		{
 			document.getElementById("sent").innerHTML = "CMetin girme";
-			//document.getElementById("result").innerHTML = "Çıktı";
 			document.getElementById("suggestion").innerHTML = "Öneri";
 			document.getElementById("Check").innerHTML = "Çek";
 			document.getElementById("Clear").innerHTML = "Berrak";
-			//document.getElementById("Confirm").innerHTML = "Onaylamak";
 			document.getElementById("lbldropdown").innerHTML = "Kullanıcı Arabirimi Dili";
 			document.getElementById("lbldropdown2").innerHTML = "Yazma Dili";
 		}
 
 	}
 
-	/**function sendData()
-	{
-		var ifinputempty = document.getElementById("Input").value;
-		if(ifinputempty !="")
-		{
-			var selectedText = document.getElementById("Input").value;
-			tempvariable = selectedText;
-			langaugetowrite=setwritinglangauge();
-
-			$.ajax({
-				url: '/',
-				type: 'POST',
-				data: { 'word': selectedText, 'language':langaugetowrite },
-			})
-				.done(function (data) {
-					Output.textContent = data;
-				});
-		}
-		else
-		{
-			alert("Please enter some text in Input");
-		}
-	}**/
-
 	function setwritinglangauge()
 	{
 		Writing_Language = document.getElementById("Writing_Language").value;
```

Comment:
Good choice of removing the OUTPUT frame. We can replace the correct words in the INPUT frame itself.